src/verifier.js

```diff
@@ -143,8 +163,22 @@ module.exports = function (options) {
 		throw new Error('Must provide both or none of --provider-states-url and --provider-states-setup-url.');
 	}
 
-	if (options.pactUrls) {
+	if (options.pactUrls && !checkTypes.emptyArray(options.pactUrls)) {
 		checkTypes.assert.array.of.string(options.pactUrls);
+		retrievePactsPromise = Promise.resolve(options.pactUrls);
```

Comment:
Yes, fair call. Actually moving it directly into the `verify()` function is even better (passing around promises here is a bit ugly). Will have an updated PR with this shortly.